lib/perl/Genome/Model/Tools/Vcf/EvaluateVcf.t

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env genome-perl
+
+# P R A G M A S ###############################################################
+use strict;
+use warnings;
+
+# M O D U L E S ###############################################################
+use Test::More;
+use Path::Class;
+use above 'Genome';
+#use Genome::Model::Tools::Vcf::EvaluateVcf;
+
+# M A I N #####################################################################
+
+#This is a bare minimum test that just compiles Perl and the UR class.
+use_ok('Genome::Model::Tools::Vcf::EvaluateVcf');
+
+SKIP: {
+    skip "this is a long-running test", 18;
+    run_evaluate_vcf();
+}
+
+done_testing();
+
+# S U B R O U T I N E S #######################################################
+sub run_evaluate_vcf {
+    my ($vcf, $gold_vcf, $roi, $true_negative_bed, $output_dir) =
+      setup_evaluate_vcf_params();
+
+    my $cmd = Genome::Model::Tools::Vcf::EvaluateVcf->create(
+        vcf               => $vcf->stringify,
+        old_sample        => "H_IJ-NA12878",
+        new_sample        => "NA12878",
+        gold_vcf          => $gold_vcf->stringify,
+        gold_sample       => "NA12878",
+        roi               => $roi->stringify,
+        true_negative_bed => $true_negative_bed->stringify,
+        output_directory  => $output_dir->stringify,
+    );
+    ok($cmd, 'Got a Genome::Model::Tools::Vcf::EvaluateVcf instance');
+    diag('executing command');
+    ok($cmd->execute, 'successfully ran execute');
+    my $stats = $cmd->rawstats;
+    ok($stats, 'got the raw statistics from vcf evaluation');
+
+    is($cmd->stat_true_positive_found_exact(),
+        28820, 'true_positive_found_exact passes');
+
+    is($cmd->stat_total_true_positive_exact(),
+        2741358, 'total_true_positive_exact passes');
+
+    is($cmd->stat_sensitivity_exact(),
+        0.0105130376988339, 'sensitivity_exact passes');
```

Comment:
The decimal statististics does match up exactly at the moment, although this test may change in the future...